pkg/operator/upgradebackupcontroller/upgradebackupcontroller.go

```diff
@@ -50,6 +51,7 @@ const (
 	recentBackupPath         = "/etc/kubernetes/cluster-backup"
 	failedPodBackoffDuration = 30 * time.Second
 	backupDirEnvName         = "CLUSTER_BACKUP_PATH"
+	featureFlagBackupAPI     = "EtcdBackupAPI"
```

Comment:
just fyi, I likely split this into another controller - that was just as an example for testing it